lms/views/admin/email.py

```diff
@@ -0,0 +1,51 @@
+from pyramid.httpexceptions import HTTPBadRequest, HTTPFound
+from pyramid.view import view_config, view_defaults
+
+from lms.security import Permissions
+from lms.tasks.email_digests import send_instructor_email_digests
+
+
+@view_defaults(route_name="admin.email", permission=Permissions.ADMIN)
+class AdminEmailViews:
+    def __init__(self, request):
+        self.request = request
+
+    @view_config(
+        route_name="admin.email",
+        request_method="GET",
+        renderer="lms:templates/admin/email.html.jinja2",
+    )
+    def get(self):
+        return {}
+
+    @view_config(route_name="admin.email", request_method="POST")
+    def post(self):
+        to_email = self.request.POST["to_email"]
+        h_userids = self.request.POST["h_userids"].split()
+        since = self.request.POST["since"]
+        until = self.request.POST["until"]
+
+        if not to_email:
+            raise HTTPBadRequest(
+                "You must enter an email address to send the test email(s) to."
+            )
```

Comment:
We wanna be really careful never to pass something like `override_to_email=None` or `""` to the celery task as this could result in emailing real users.

Reply:
We might need to be defensive about it on the actual service (eg check against an explicit bool, not just truthy values).

Reply:
Yep, we should be defensive about this in the service itself as well